From 59b1cb227364e783214bbed82e30ae8ece240b67 Mon Sep 17 00:00:00 2001
From: Stephan Hartmann <stha09@googlemail.com>
Date: Fri, 25 Feb 2022 07:25:15 +0000
Subject: [PATCH] libstdc++: fix ostream operator<< in CameraIndicationObserver

std::unique_ptr has no operator<< in libstdc++ before C++20. Use
get() to retrieve the pointer value.
---
 chrome/browser/vr/chrome_xr_integration_client.cc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/chrome/browser/vr/chrome_xr_integration_client.cc b/chrome/browser/vr/chrome_xr_integration_client.cc
index 74ee91e..ef1aade 100644
--- a/chrome/browser/vr/chrome_xr_integration_client.cc
+++ b/chrome/browser/vr/chrome_xr_integration_client.cc
@@ -48,7 +48,7 @@ class CameraIndicationObserver : public content::BrowserXRRuntime::Observer {
                                bool in_use) override {
     DVLOG(3) << __func__ << ": web_contents=" << web_contents
              << ", in_use=" << in_use << ", num_runtimes_with_camera_in_use_="
-             << num_runtimes_with_camera_in_use_ << ", ui_=" << ui_;
+             << num_runtimes_with_camera_in_use_ << ", ui_=" << ui_.get();
     // If `in_use` is true, we need to have a non-null `web_contents` to be able
     // to register the media stream:
     DCHECK(!in_use || web_contents);
-- 
2.34.1

